app/assets/stylesheets/global.css.scss

```diff
@@ -22,9 +22,14 @@
   background-color: rgba(0, 163, 255, .0)
 }
 
+.menu-text {
+  font-size: 1.8rem;
+  padding: 0 1.2rem;
+}
+
 .border {
   height: .2rem;
-  background-color: #69F1FE;
+  background-color: #69f1fe;
```

Comment:
Color literals like `#69f1fe` should only be used in variable declarations; they should be referred to via variable everywhere else.